### PR TITLE
Feature/39713 - Minicart

### PR DESCRIPTION
**Pull request by Aekal**
Internal: 39713

### CODE REVIEW

File: Magento_Checkout/web/template/minicart/content.html [hunk 1]
```diff
@@ -1,37 +1,12 @@
...
+<div class="minicart-content__total">
+    <!-- ko ifnot: getCartParam('summary_count') -->
+        <div class="subtitle empty" data-bind="visible: closeSidebar()">
+            <!-- ko i18n: 'You have no items in your shopping cart.' --><!-- /ko -->
+        </div>
+    <!-- /ko -->
     <!-- ko if: getCartParam('summary_count') -->
-        <div class="items-total">
-            <span class="count"><!-- ko text: getCartParam('summary_count') --><!-- /ko --></span>
+        <div class="minicart-content__total-counter items-total">
```
Comment (szafran89) [inline Magento_Checkout/web/template/minicart/content.html]: check if we can remove `items-total` class

File: Magento_Checkout/web/template/minicart/item/default.html [hunk 2]
```diff
@@ -1,99 +1,166 @@
...
+                            title: $t('Update')
+                        }
+                    "
+                    class="
+                        button
+                        button--secondary
+                        minicart-product__update
+                        update-cart-item
+                    "
+                    style="display: none"
```
Comment (szafran89) [inline Magento_Checkout/web/template/minicart/item/default.html]: can we move inline styles to sass file?

Comment (szafran89) [reply Magento_Checkout/web/template/minicart/item/default.html]: DO we need class `update-cart-item`?

Comment (szafran89) [reply Magento_Checkout/web/template/minicart/item/default.html]: Missing button type

File: Magento_Checkout/web/template/minicart/item/default.html [hunk 3]
```diff
@@ -1,99 +1,166 @@
...
-                       class="action edit button button--icon">
-                        <svg class="button__icon">
-                            <use data-bind="attr: { 'xlink:href': require.toUrl('images/icons-sprite.svg#edit') }" xlink:href='' ></use>
+                    <a
+                        data-bind="attr: {href: configure_url, title: $t('Edit item')}"
+                        class="button button--icon minicart-product__button"
+                        aria-label="edit product in your shopping cart"
+                    >
+                        <svg
+                            title="edit"
```
Comment (szafran89) [inline Magento_Checkout/web/template/minicart/item/default.html]: we don't need title attribute in svg tag

File: Magento_Checkout/web/template/minicart/item/default.html [hunk 4]
```diff
@@ -1,99 +1,166 @@
...
-                       class="action delete button button--icon"
+
+                <a
+                    href="#"
+                    data-bind="attr: {'data-cart-item': item_id, title: $t('Remove item')}"
+                    class="button button--icon minicart-product__button action delete"
+                    aria-label="remove this product from your shopping cart"
+                >
+                    <svg
+                        title="remove"
```
Comment (szafran89) [inline Magento_Checkout/web/template/minicart/item/default.html]: we don't need title attribute in svg tag

File: Magento_Checkout/web/template/minicart/subtotal.html [hunk 5]
```diff
@@ -1,14 +1,7 @@
...
- */
--->
-<div class="subtotal grid minicart__subtotal">
-    <span class="label grid--two-cols">
-        <!-- ko i18n: 'Cart Subtotal' --><!-- /ko -->
-    </span>
-
+<div class="minicart-content__summary">
+    <div class="minicart-content__summary-label subtotal">
+        <span class="price"><!-- ko i18n: 'Cart Subtotal' --><!-- /ko --></span>
```
Comment (szafran89) [inline Magento_Checkout/web/template/minicart/subtotal.html]: move `<!-- ko i18n: 'Cart Subtotal' --><!-- /ko -->` to new line

File: Magento_Checkout/web/template/minicart/content.html (hunk 1 above)
Comment (Aekal) [reply Magento_Checkout/web/template/minicart/content.html]: Removed

File: Magento_Checkout/web/template/minicart/item/default.html (hunk 2 above)
Comment (Aekal) [reply Magento_Checkout/web/template/minicart/item/default.html]: Done

File: Magento_Checkout/web/template/minicart/item/default.html (hunk 3 above)
Comment (Aekal) [reply Magento_Checkout/web/template/minicart/item/default.html]: Removed

File: Magento_Checkout/web/template/minicart/item/default.html (hunk 4 above)
Comment (Aekal) [reply Magento_Checkout/web/template/minicart/item/default.html]: Removed

File: Magento_Checkout/web/template/minicart/subtotal.html (hunk 5 above)
Comment (Aekal) [reply Magento_Checkout/web/template/minicart/subtotal.html]: Done

**Pull request merged**